FileDirectoryManager/src/org/jjazz/filedirectorymanager/api/FileDirectoryManager.java

```diff
@@ -149,14 +149,23 @@ public File getNewSongTemplateSongFile()
     public File getJJazzLabUserDirectory()
     {
         // We need a valid user.home directory        
-        String uh = System.getProperty("user.home");
+        final String uh = System.getProperty("user.home");
         assert uh != null : "user.home property does not exist: " + uh;   //NOI18N
         File userHome = new File(uh);
         assert userHome.isDirectory() : "user.home directory does not exist: " + uh;   //NOI18N
 
+        final String dataDir;
+        if (System.getProperty("os.name").equalsIgnoreCase("linux")) {
```

Comment:
The Netbeans platform has a generic isUnix() : https://bits.netbeans.org/12.4/javadoc/org-openide-util/org/openide/util/BaseUtilities.html#isUnix--

Reply:
I actually don't think MacOS follows the XDG standard (and according to the docs, Netbeans returns `true` on MacOS for `isUnix()`.  It's been a while since I've used a fruity machine, but I don't think there's a `.local` folder there.

So, I think it's safer to do linux-only here (and perhaps BSD, if we can find someone running that :) )